eng/pipelines/cadl-ci.yaml

```diff
@@ -135,3 +135,11 @@ jobs:
         artifact: CoverageReport
         displayName: 'Publish Cadl Ranch Coverage Report'
 
+      - task: AzureCLI@2
+        displayName: 'Upload to Cadl Ranch Coverage Report'
+        inputs:
+          azureSubscription: "Azure SDK Playground"
+          scriptType: "bash"
+          scriptLocation: "inlineScript"
+          inlineScript: cadl-ranch upload-coverage --coverageFile ./cadl-ranch-coverage-java.json --generatorName java --storageAccountName cadlranchcoverage --generatorVersion 0.1.0
```

Comment:
`--generatorVersion 0.1.0`

just a note to make sure to retrieve the real version for this

Reply:
Sure, I will update